explorer/client/cypress/localnet.ts

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2022, Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import axios from 'axios';
+
+// NOTE: We import out of the source directory here to work around an issue with Cypress not
+// respecting tsconfig paths in the config file.
+import { Ed25519Keypair, type Keypair } from '../../../sdk/typescript/src';
+
+export async function createLocalnetTasks() {
+    const addressToKeypair = new Map<string, Keypair>();
+
+    return {
+        async faucet() {
+            const keypair = Ed25519Keypair.generate();
+            const address = keypair.getPublicKey().toSuiAddress();
+            addressToKeypair.set(address, keypair);
+            const res = await axios.post<{ ok: boolean }>(
+                'http://localhost:9123/faucet',
```

Comment:
Works for me locally after this change
```suggestion
                'http://127.0.0.1:9123/faucet',
```